src/main/java/slimeknights/tconstruct/smeltery/data/SmelteryRecipeProvider.java

```diff
@@ -193,6 +194,89 @@ private void addBaseRecipes(Consumer<IFinishedRecipe> consumer) {
+      .addIngredient(Items.IRON_INGOT)
+      .addIngredient(TinkerTags.Items.BLOOD_SLIMEBALL).addIngredient(TinkerTags.Items.BLOOD_SLIMEBALL)
+      .addCriterion("has_item", hasItem(TinkerSmeltery.searedBrick))
+      .build(consumer, wrap(TinkerMaterials.pigironIngot.get(), folder, "_temporary"));
+
+    // TODO: Temporary Rose Gold Recipe
+    ShapelessRecipeBuilder.shapelessRecipe(TinkerMaterials.roseGoldIngot.get(), 3)
+      .addIngredient(Items.GOLD_INGOT)
+      .addIngredient(TinkerTags.Items.INGOTS_COPPER).addIngredient(TinkerTags.Items.INGOTS_COPPER)
+      .addCriterion("has_item", hasItem(TinkerSmeltery.searedBrick))
+      .build(consumer, wrap(TinkerMaterials.roseGoldIngot.get(), folder, "_temporary"));
+
+    // TODO: Temporary Soulsteel Recipe
+    ShapelessRecipeBuilder.shapelessRecipe(TinkerMaterials.soulSteelIngot.get(), 3)
+      .addIngredient(TinkerTags.Items.INGOTS_COBALT)
+      .addIngredient(ItemTags.SOUL_FIRE_BASE_BLOCKS).addIngredient(TinkerTags.Items.INGOTS_SLIME_STEEL)
+      .addCriterion("has_item", hasItem(TinkerSmeltery.searedBrick))
+      .build(consumer, wrap(TinkerMaterials.soulSteelIngot.get(), folder, "_temporary"));
+
+    // TODO: Temporary Alexandrite Recipe
+    ShapelessRecipeBuilder.shapelessRecipe(TinkerMaterials.alexandritePearl.get(), 1)
+      .addIngredient(Items.ENDER_EYE)
+      .addCriterion("has_item", hasItem(TinkerSmeltery.searedBrick))
+      .build(consumer, wrap(TinkerMaterials.alexandritePearl.get(), folder, "_temporary"));
+
+    // TODO: Temporary Heptazion Recipe
+    ShapelessRecipeBuilder.shapelessRecipe(TinkerMaterials.heptazionIngot.get(), 3)
+      .addIngredient(TinkerTags.Items.INGOTS_COBALT)
+      .addIngredient(Blocks.OBSIDIAN).addIngredient(TinkerTags.Items.INGOTS_COPPER)
+      .addCriterion("has_item", hasItem(TinkerSmeltery.searedBrick))
+      .build(consumer, wrap(TinkerMaterials.heptazionIngot.get(), folder, "_temporary"));
+
+    // TODO: Temporary Slimebronze Recipe
+    ShapelessRecipeBuilder.shapelessRecipe(TinkerMaterials.slimeBronzeIngot.get(), 3)
+      .addIngredient(TinkerTags.Items.INGOTS_COBALT)
+      .addIngredient(TinkerTags.Items.MAGMA_SLIMEBALL).addIngredient(TinkerTags.Items.INGOTS_BRONZE)
+      .addCriterion("has_item", hasItem(TinkerSmeltery.searedBrick))
+      .build(consumer, wrap(TinkerMaterials.slimeBronzeIngot.get(), folder, "_temporary"));
+
+    // TODO: Temporary Manyullyn Recipe
+    ShapelessRecipeBuilder.shapelessRecipe(TinkerMaterials.manyullynIngot.get(), 2)
+      .addIngredient(TinkerTags.Items.INGOTS_COBALT)
+      .addIngredient(TinkerTags.Items.INGOTS_ARDITE).addIngredient(TinkerTags.Items.INGOTS_ARDITE)
+      .addCriterion("has_item", hasItem(TinkerSmeltery.searedBrick))
+      .build(consumer, wrap(TinkerMaterials.manyullynIngot.get(), folder, "_temporary"));
+
+    // TODO: Temporary Knightslime Recipe
+    ShapelessRecipeBuilder.shapelessRecipe(TinkerMaterials.knightslimeIngot.get(), 2)
```

Comment:
We have another temporary knightslime recipe in another module. Make sure to clean those up.